doc/rst/language/spec/arrays.rst

```diff
@@ -237,11 +256,38 @@ procedure with an implicit return type (see
 
 ..
 
-   *Open issue*.
+   *Example (adecl-2x2x3-literal.chpl)*.
+
+   The following example declares a 2x2x3 element rectangular array literal
```

Comment:
Nitty, and I realize you're following the stylization of the previous example(s), but I'd hyphenate `2x2x3-element` since it acts as an adjective in total.